develop/addons/schema-driven-forms/customising-form-behaviour/actions.rst

```diff
@@ -27,7 +27,7 @@ in our pizza order form:
         def handleCancel(self, action):
             ...
 
-The name of the method is not particularly important, so long as it is
+The name of the method is not particularly important, it only needs to be
```

Comment:
On 05/17/2016 04:05 PM, Stephan Klinger wrote:
>
> In
> develop/addons/schema-driven-forms/customising-form-behaviour/actions.rst
> <https://github.com/plone/documentation/pull/611#discussion_r63527537>:
>
> > @@ -27,7 +27,7 @@ in our pizza order form:
> >          def handleCancel(self, action):
> >              ...
> >  
> > -The name of the method is not particularly important, so long as it is
> > +The name of the method is not particularly important, it only needs to be
>
> What about 'but it needs to be unique.'?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/plone/documentation/pull/611/files/8268ac35ed2a073e7d4e11b152e4ea2f8952691d#r63527537>
>
This is fine ! :)

We are want to avoid words like: just, only, easy, ... because the could
give user a wrong impression, what may easy for you is maybe not that
easy for me :)

If you are person ho tries to get all this stuff, such words can be a
huge burden, that is why we want to avoid them.